sql/000_base_structure.sql

```diff
@@ -248,6 +248,12 @@ CREATE TABLE `statistics_users` (
   KEY `account_id_timestamp` (`account_id`,`timestamp`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE IF NOT EXISTS `push_notification_settings` (
```

Comment:
Maybe this could be a more generic user_settings table? I remember that I wanted to implement this in the past to allow for a extraction of per-user settings from the accounts table to a settings table. Just an idea, maybe this is sufficient.